breaking-changes.md

```diff
@@ -12,17 +12,14 @@ permalink: /breaking-changes/
 
 The `type` parameter has been removed from all OpenSearch API endpoints. Instead, indexes can be categorized by document type. For more details, see issue [#1940](https://github.com/opensearch-project/opensearch/issues/1940).
 
-### Deprecate outdated nomenclature 
+### Deprecate non-inclusive terms 
 
-In order for OpenSearch to include more inclusive naming conventions, we've replaced the following terms in our code with more inclusive terms:
+Non-inclusive terms have been deprecated in the 2.x releases and will be permanently removed in OpenSearch 3.0.  We are using the following replacements: 
```

Comment:
Passive voice ("...have been deprecated.."). Change to something like:

```
Non-inclusive terms are deprecated in version 2.x and will not be supported with the next major release.
```

Reply:
Thanks, Jeff! A previous consensus was to say 'permanently removed' to express the seriousness of this change. Of course, this will be documented for 3.0 as a breaking change.